sound/soc/intel/boards/sof_sdw.c

```diff
@@ -424,6 +446,12 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.dai_name = "rt1316-aif",
 		.init = sof_sdw_rt1316_init,
 	},
+	{
+		.part_id = 0x1319,
+		.direction = {true, false},
```

Comment:
true, true?

there should be support for I/V feedback as done for 1316, no?

Reply:
@plbossart Thank you for pointing out the problem. 
Although the codec name is 1319, its device id may be 1316(This is the case on SKU 0AF3 product). If so, the 1319 code added in this PR will be removed. I am contacting Alex in dell to make sure to get the new information of other four product, and then update the PR.